### ISSUE TITLE
Loading data should really strip spaces and handle quotes properly.  

### ISSUE THREAD

**Issue by lisad** [lisad/phaser#54]
This test demonstrates the problem: 
```

@pytest.mark.skip("This is annoying.  it's easy for CSVs to have quotes and spaces, but the combination is bad.")
def test_load_quoted_values(tmpdir):
    source = tmpdir / 'quoted_values.csv'
    with open(source, 'w') as f:
        # Importantly, Active is quoted AND has a space before it.
        f.write("""id,name,status\n1,"Jean-Luc Picard", "Active"\n""")
    phase= Phase()
    phase.load(source)
    # If values are not stripped of space and converted, status = ' "Active"'
    assert phase.row_data[0]['status'] == "Active"

```

If this isn't fixed, it's all too easy to have a bug in the pipeline.  I encountered this in the employees.csv and employees pipeline work, where paidPer = ' "Year"' instead of 'Year', so the code failed to calculate salary correctly.

**Comment by jeffkole** [comment]
Likely fixed as part of #78

**Comment by lisad** [comment]
This is now addrssed where Phase does column name canonicalization.
Also I'm leaving the test in the io/csv test section because if this ever changes
we'd like to know.
